app/views/key_pairs/index.html.erb

```diff
@@ -0,0 +1,53 @@
+<% title _("Key pairs for: %s") % @compute_resource.name %>
+
+<table class="<%= table_css_classes 'table-fixed' %>">
+  <% if @key_pairs.any? %>
+  <thead>
+  <tr>
+    <th width="6%"><%= _('Active key') %></th>
+    <th width="40%"><%= _('Name') %></th>
```

Comment:
Like that?
![image](https://cloud.githubusercontent.com/assets/433583/20882986/66357eec-baec-11e6-91e2-2149306153ba.png)